coursedata/adventures/de.yaml

```
@@ -320,71 +320,71 @@ adventures:
                     ```
                 start_code: ""
     parrot:
-        name: "Parrot"
-        description: "Create your own online pet parrot that will copy you!"
+        name: "Papagei"
+        description: "Erschaffe Dir deinen eigenen zahmen Papageien, der Dir alles nachplappert!"
         image: "story.png"
-        default_save_name: "Parrot"
+        default_save_name: "Papagei"
         levels:
             1:
                 story_text: |
-                    ## Parrot
-                    Create your own online pet parrot that will copy you!
+                    ## Papagei
+                    Erschaffe Dir deinen eigenen zahmen Papageien, der Dir alles nachplappert!
 
-                    ## Example Hedy code
+                    ## Hedy Beispiel-Code
                     ```
-                    print Im Hedy the parrot
-                    ask whats your name?
+                    print Ich bin Hedy der Papagei!
+                    ask Wie lautet Dein Name?
                     echo
                     echo
                     ```
-                start_code: "print Im Hedy the parrot"
+                start_code: "print Ich bin Hedy der Papagei!"
             2:
                 story_text: |
-                    ## Parrot
-                    Create your own online pet parrot that will copy you!
+                    ## Papagei
+                    Lass Deinen zahmen Papageien noch etwas echter klingen!
 
-                    ## Example Hedy code
+                    ## Hedy Beispiel-Code
```

I know you are translating from the current English version, so you are not to blame for this, but we are in the middle of making a change to the yaml format, almost ready to be merged (see #1838), where example code will be explicitly labeled with a key in yaml, like this:

```suggestion
                example_code: |
                    ## Hedy Beispiel-Code
```

I can totally fix that for this PR of course, but if you go about translating new ones, maybe you can stick to this new format already just to save me a bit of work? Thanks!!

---

Hey,
alright, will take care of it for the next PR.